blogs/network_events/index.md

```diff
@@ -0,0 +1,202 @@
+# <div align="center">Monitoring OVN Networking Events using Network Observability</div>
+
+<p align="center">
+  <img src="./images/network_events_logo.drawio.png" alt="logo" width="25%"/>
+</p>
+
+By: Mohamed S. Mahmoud
+
+Network security and performance maintenance crucially depend on monitoring network events triggered
+by OVN (Open Virtual Network), including network policies, admin policies, and egress firewalls.
+You can achieve this using network observability eBPF agent, which runs in the Linux kernel and can trace
+various network activities with minimal performance overhead, allowing you to observe and capture detailed
+information about network traffic and events in real-time.
+
+## Key Components for Network Observability with eBPF
+
+1- eBPF network events monitoring `Kprobe` eBPF hook
+
+Network events monitoring using eBPF kernel probes (kprobes) provides deep, kernel-level insights into
+network stack behavior. The NetObserv eBPF agent leverages the [psample_sample_packet](https://github.com/torvalds/linux/blob/e32cde8d2bd7d251a8f9b434143977ddf13dcec6/net/psample/psample.c#L363) entry point to efficiently
+capture packet metadata and identify policy violations with minimal overhead.
+
+The hook implemented in the Netobserv eBPF agent can capture multiple events within the same network flow
```

Comment:
For consistency, use the same capitalization across the doc (and NetObserv is how we call it upstream)

```suggestion
The hook implemented in the NetObserv eBPF agent can capture multiple events within the same network flow
```